src/models/multiBarChart.js

```diff
@@ -429,8 +674,25 @@ nv.models.multiBarChart = function() {
             rightAlignYAxis = _;
             yAxis.orient( rightAlignYAxis ? 'right' : 'left');
         }},
+        xTickFormat: {get: function(){return xAxis.xTickFormat();}, set: function(_){
+            xAxis.xTickFormat(_);
+            x2Axis.xTickFormat(_);
+        }},
+        yTickFormat: {get: function(){return yAxis.yTickFormat();}, set: function(_){
+            yAxis.yTickFormat(_);
+            y2Axis.yTickFormat(_);
+        }},
+        x: {get: function(){return multibar.x();}, set: function(_){
+            multibar.x(_);
+            multibar2.x(_);
+        }},
+        y: {get: function(){return multibar.y();}, set: function(_){
+            multibar.y(_);
+            multibar2.y(_);
+        }},
         barColor:  {get: function(){return multibar.barColor;}, set: function(_){
             multibar.barColor(_);
+            multibar2.barColor(_);
             legend.color(function(d,i) {return d3.rgb('#ccc').darker(i * 1.5).toString();})
```

Comment:
This was missed after certain trial/errors. Thanks for pointing it out. Please confirm earlier to changes as well, so I can add the commit to PR.